### PR TITLE
Add Help menu for Korman.

### PR DESCRIPTION
**Pull request by Deledrius**
This adds a Help menu to the appropriate place in Blender.  It provides links to common documents of use to Age builders.

### CODE REVIEW

File: korman/ui/ui_menus.py [hunk 1]
```diff
@@ -40,8 +40,27 @@ def build_plasma_menu(self, context):
...
+class korman_help_menu(bpy.types.Menu):
+    bl_idname = "menu.plasma_help"
+    bl_label = "Korman..."
+
+    def draw(self, context):
+        layout = self.layout
+
+        layout.operator("wm.url_open", text="About Korman", icon='URL').url="https://guildofwriters.org/wiki/Korman"
+        layout.operator("wm.url_open", text="Getting Started", icon='URL').url="https://guildofwriters.org/wiki/Korman:Getting_Started"
+        layout.operator("wm.url_open", text="Tutorials", icon='URL').url="https://guildofwriters.org/wiki/Category:Korman_Tutorials"
```
Comment (Hoikas) [inline korman/ui/ui_menus.py:53]: Whitespace nit. `str` should be double quoted `"`

**Comment by Deledrius** [comment]
Additional page suggestions for links are welcome.  I think it would also be great to have a link to a page that allows the user to easily check for updates, but that would require adding a page to the website for this purpose.

File: korman/ui/ui_menus.py [hunk 2]
```diff
@@ -40,8 +40,27 @@ def build_plasma_menu(self, context):
...
+
+
+def build_korman_help_menu(self, context):
+    self.layout.menu("menu.plasma_help", text="Korman", icon='URL')
```
Comment (Hoikas) [inline korman/ui/ui_menus.py:57]: `str` should be double quoted `"`

File: korman/ui/ui_menus.py [hunk 3]
```diff
@@ -40,8 +40,27 @@ def build_plasma_menu(self, context):
...
+
+
```
Comment (Hoikas) [inline korman/ui/ui_menus.py:59]: According to PEP8, functions shall be followed by only one blank line.

File: korman/ui/ui_menus.py (hunk 2 above)
Comment (Deledrius) [reply korman/ui/ui_menus.py:57]: Copypasta errors. >_<

File: korman/ui/ui_menus.py [hunk 4]
```diff
@@ -40,8 +40,25 @@ def build_plasma_menu(self, context):
...
+class korman_help_menu(bpy.types.Menu):
+    bl_idname = "menu.plasma_help"
+    bl_label = "Korman..."
+
+    def draw(self, context):
+        layout = self.layout
+        layout.operator("wm.url_open", text="About Korman", icon="URL").url="https://guildofwriters.org/wiki/Korman"
+        layout.operator("wm.url_open", text="Getting Started", icon="URL").url="https://guildofwriters.org/wiki/Korman:Getting_Started"
+        layout.operator("wm.url_open", text="Tutorials", icon="URL").url="https://guildofwriters.org/wiki/Category:Korman_Tutorials"
```
Comment (Hoikas) [inline korman/ui/ui_menus.py:52]: Whitespace

**Comment by Deledrius** [comment]
That depends!  Are there any other obvious links we should include?

**Comment by Hoikas** [comment]
I can't think of anything else right now.

**Comment by DoobesURU** [comment]
I say the simpler, the better. This should be enough.

**Comment by Deledrius** [comment]
There are things I want to add, but they can be additions.  This is a good baseline.

File: korman/ui/ui_menus.py [hunk 5]
```diff
@@ -40,8 +40,25 @@ def build_plasma_menu(self, context):
     self.layout.separator()
     self.layout.menu("menu.plasma_add", icon="URL")
 
+
+class korman_help_menu(bpy.types.Menu):
```
Comment (Hoikas) [inline korman/ui/ui_menus.py:44]: The class name is not consistent with PEP8, and we need to inherit from `PlasmaMenu`.

**Comment by Deledrius** [comment]
I'd like to change `build_plasma_menu` to be `build_plasma_add_menu` for consistency's sake.  Should I piggy-back that on here with another commit, or open a whole PR just for that?

**Review by Hoikas — APPROVED** [review]

**Comment by Hoikas** [comment]
Let's hold off until another PR on that. It would make more sense to me if those were a `build_menu` method on the menu classes, but meh.

**Pull request merged**